.env.example

```diff
@@ -1,11 +1,16 @@
-BOT_NAME=
-BOT_AUTH=
-CHANNELS=
-CLIENT_ID=
-CLIENT_SECRET=
-PORT=
-TWITCH_API_CALLBACK_URL=
-ALLOWED_CALLBACK_URLS=https://p4nth3rb0t-mainframe.herokuapp.com,https://theclaw.dev
-DB_CONN_STRING=
-NODE_ENV=
-DISCORD_TOKEN=
\ No newline at end of file
+BOT_NAME=
+BOT_AUTH=
+CHANNELS=
+CLIENT_ID=
+CLIENT_SECRET=
+PORT=
+TWITCH_API_CALLBACK_URL=
+ALLOWED_CALLBACK_URLS=https://p4nth3rb0t-mainframe.herokuapp.com,https://theclaw.dev
+DB_CONN_STRING=
+NODE_ENV=
+DISCORD_TOKEN=
+
+# This requires the "repo" scope.
+GITHUB_ACCESS_TOKEN=
+# probably?
+GITHUB_USERNAME=whitep4nth3r

```

Comment:
I don't think all the user gists should go into Salma's personal GitHub Gists, to not spam her own profile with them.

Which, I guess, means the bot needs its own GitHub organisation now.

Reply:
I'm not sure if organizations can have gists but there's nothing stopping `$communityManager` from creating a "theclawcommunity" account.